src/CMakeLists.txt

```diff
@@ -33,6 +33,11 @@ include_directories(
     ${CMAKE_CURRENT_SOURCE_DIR}/cpu/xbyak
     )
 
+if(MKLDNN_DISABLE_JIT)
+    add_definitions(-DMKLDNN_DISABLE_JIT)
```

Comment:
I don't think that this define is really necessary.

Reply:
Agree, will change it.